setup.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
This `#!` is required.  Just a reminder not to delete it.

Reply:
Ok will remove it.